src/compiler/program.ts

```diff
@@ -241,22 +241,28 @@ namespace ts {
         return errorMessage;
     }
 
-    const redForegroundEscapeSequence = "\u001b[91m";
-    const yellowForegroundEscapeSequence = "\u001b[93m";
-    const blueForegroundEscapeSequence = "\u001b[93m";
+    /** @internal */
+    export const foregroundColorEscapeSequences = {

```

Comment:
This can just be a const string enum `ColorEscapeSequence`, and it doesn't need to be exported at all.